input/europe/eu.csv

```diff
@@ -0,0 +1,150 @@
+Country,Name,Reactor no.,Reactor,Status,Capacity in MW,Construction start date,Commercial operation date,Closure (planned),,
```

Comment:
I'd like to generally recommend that column names are easier to handle in the long run if they are very simple with no spaces or special characters. 

The changes I would make:
- Reactor no. -> "Unit" . I assume "Reactor no." refers to the unit number of a reactor which shares a site with others. Is that the case? If so, I might change the heading to, simply, "Unit". 
- Capacity in MW -> MWe (I assume you mean MWe. If you mean MWth, then that should be stated instead). 
- Construction start date -> "Construction" or "StartDate" or "ConstructionStartDate"
- Commercial Operation date -> "OperationStart"
- Closure (planned) -> Closure